pkg/apis/toolchain/v1alpha1/usersignup_types.go

```diff
@@ -33,11 +33,14 @@ type UserSignupSpec struct {
 	// +optional
 	Approved bool `json:"approved,omitempty"`
 
-	// The username.  This may differ from the UserSignup's metadata.name, which is restricted by the
-	// limited character set available for naming (see RFC1123).  If the username contains characters which are
-	// disqualified from the resource name, the username is transformed into an acceptable resource name instead.
-	// For example, johnsmith@redhat.com -> johnsmith-at-redhat-com
+	// The user's username, obtained from the identity provider.
 	Username string `json:"username"`
+
+	// The DNS-1123-compliant username.  This username is generated by the registration app and may differ from the
+	// Username, as a limited character set is available for naming (see RFC1123).  If the username contains characters
+	// which are disqualified, the username is transformed into a compliant name instead.
+	// For example, johnsmith@redhat.com -> johnsmith-at-redhat-com
+	CompliantUsername string `json:compliantUsername`
```

Comment:
```suggestion
	CompliantUsername string `json:"compliantUsername"`
```

Don't forget to regenerate everything after changing it.

Reply:
Fixed this